src/web/screens/sharingPermissions.tsx

```diff
@@ -152,9 +151,4 @@ export const SharingPermissionsRoute: PortalRoute = {
   description: 'Sharing Permissions',
   element: <SharingPermissions />,
   path: '/dashboard/sharing',
-  loader: () => {
-    preloadAvailableSiteList();
```

Comment:
Why exactly are we removing these?

Reply:
Is it because we are calling it on `home.tsx` so it is just unnecessary now?

Reply:
They no longer seemed to be in use.  We were just sending a request for the data and never doing anything with it.